fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -190,6 +190,9 @@ public enum ConnectType {
 
     private TResultSinkType resultSinkType = TResultSinkType.MYSQL_PROTOCAL;
 
+    //internal call like `insert overwrite` not need skipAuth
```

Comment:
The commend is misleading?
you can give a example in comment to describe how to use this `skipAuth`